crypto/src/lib.rs

```diff
@@ -16,5 +16,17 @@
 // attribute macro and an order of operations well defined
 
 pub mod base58;
+pub mod base64;
+pub mod binprot;
 pub mod hash;
+pub mod hex;
 pub mod signature;
+
+pub mod prelude {
```

Comment:
I think it is best practice to only export traits in a prelude so consider removing the hash and signature types?

Reply:
fixed